A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/Airbus_FMA.js

```diff
@@ -1519,12 +1527,12 @@ var Airbus_FMA;
         }
         updateRow2(_deltaTime) {
             var targetState = Column4.ROW_2_STATE.NONE;
-            if (Simplane.getAutoPilotAPPRActive() && Airbus_FMA.CurrentPlaneState.isILSApproachActive && Airbus_FMA.CurrentPlaneState.anyAutoPilotsActive) {
-                if (Airbus_FMA.CurrentPlaneState.bothAutoPilotsActive) {
+            if (isCAT3Capable == true) {
```

Comment:
`== true` is not needed.

```suggestion
            if (isCAT3Capable) {
```